python/ray/dashboard/client/src/common/timezone.ts

```diff
@@ -1,794 +1,691 @@
-export const timezones = [
+const getCurrentUTCOffset = (timeZone: string) => {
+  try {
+    const date = new Date();
+    // Ideally this would be longOffset, but it's not supported in used version of node
+    const formatter = new Intl.DateTimeFormat("en-US", {
+      timeZone,
+      timeZoneName: "shortOffset",
+    });
+    // Extract the GMT offset part from the formatted string and ensure it has minutes
+    const formatted = formatter.format(date);
+    const match = formatted.match(/GMT[+-]\d{1,2}(?::\d{2})?/);
+    if (match) {
+      // Ensure the offset has minutes and hours are padded (e.g., convert GMT+5 to GMT+05:00)
+      const offset = match[0];
+      if (!offset.includes(":")) {
+        return offset.replace(
+          /([+-])(\d{1,2})$/,
+          (_, sign, hours) => `${sign}${hours.padStart(2, "0")}:00`,
+        );
+      }
+      return offset.replace(
+        /([+-])(\d{1,2}):/,
+        (_, sign, hours) => `${sign}${hours.padStart(2, "0")}:`,
```

Comment:
I don't think we need the to re-format the string to look like longOffset. Let's just use the shortOffset format text in the UI.

Reply:
Will look into- need to trace through how it's used within `SearchComponent.tsx`

Reply:
Confirmed change to shortOffset is benign as it's only used for display. Time zone name is what drives the actual display using dayjs.